wgpu-types/src/features.rs

```diff
@@ -1509,6 +1509,19 @@ impl Features {
         ]))
     }
 
+    /// Mask of all features which are experimental.
+    #[must_use]
+    pub const fn all_experimental_mask() -> Self {
+        Self::from_bits_truncate(FeatureBits([
+            FeaturesWGPU::EXPERIMENTAL_MESH_SHADER.bits()
+                | FeaturesWGPU::EXPERIMENTAL_MESH_SHADER_MULTIVIEW.bits()
+                | FeaturesWGPU::EXPERIMENTAL_RAY_QUERY.bits()
+                | FeaturesWGPU::EXPERIMENTAL_RAY_HIT_VERTEX_RETURN.bits()
+                | FeaturesWGPU::EXPERIMENTAL_PASSTHROUGH_SHADERS.bits(),
```

Comment:
Couldn't this be simplified?

```suggestion
            (FeaturesWGPU::EXPERIMENTAL_MESH_SHADER
                | FeaturesWGPU::EXPERIMENTAL_MESH_SHADER_MULTIVIEW
                | FeaturesWGPU::EXPERIMENTAL_RAY_QUERY
                | FeaturesWGPU::EXPERIMENTAL_RAY_HIT_VERTEX_RETURN
                | FeaturesWGPU::EXPERIMENTAL_PASSTHROUGH_SHADERS
            ).bits(),
```

Reply:
Not while keeping the function const

Reply:
Oh, right, it would have to be `.union()`. Probably not worth changing in that case.